pkg/monitor/monitor/util.go

```diff
@@ -729,3 +734,10 @@ func getMonitorPVC(monitor *v1alpha1.TidbMonitor) *core.PersistentVolumeClaim {
 		},
 	}
 }
+
+// If the namespace in ClusterRef is empty, we would set the TidbMonitor's namespace in the default
+func setDefaultNamespaceForClusterRef(model *MonitorConfigModel, tm *v1alpha1.TidbMonitor) {
```

Comment:
I think we don't need to create a new function for this

Reply:
updated.